engine/launcher/general.cpp

```diff
@@ -126,7 +126,7 @@ char *replace(char *line, char *search, char *replace, int LENGTH) {
 	current= new char [LENGTH+1];
 	length = strlen(line);
 	strcpy(current, line);
-	while ((location = strstr(current, search)) > 0) {
+	while ((location = strstr(current, search)) != NULL) {
```

Comment:
we may want to use `std::nullptr` (https://en.cppreference.com/w/cpp/language/nullptr) instead of NULL. Not sure if that's good for this PR or not, so not blocking but something to start thinking about.

Reply:
Due to @royfalk's comment, I think that such a change should be brought up for discussion, and should be done in a separate PR after some testing.